src/gui/widgets/InstrumentMidiIOView.cpp

```diff
@@ -84,8 +84,9 @@ InstrumentMidiIOView::InstrumentMidiIOView( QWidget* parent ) :
 	midiOutputLayout->setSpacing( 6 );
 
 	m_outputChannelSpinBox = new LcdSpinBox( 2, m_midiOutputGroupBox );
+	m_outputChannelSpinBox->addTextForValue( 0, "--" );
 	m_outputChannelSpinBox->setLabel( tr( "CHANNEL" ) );
-	m_outputChannelSpinBox->setEnabled( false );
+	m_outputChannelSpinBox->setEnabled( true );
```

Comment:
What if we don't want to enable midi output? This will always be enabled then.

Reply:
That only enables the midi output channel spin box widget, not the midi output itself. That's so you're able to choose the channel that the track will use to send the midi events to the instrument plugin without needing to enable midi output.

Reply:
Aren't `LcdSpinBox`es enabled by default? In which case, this line could be removed altogether.

Reply:
You're right @DomClark , I was making the requested changes and I'm about to update the PR.